2.11: Upgrade Nvidia Drivers, CUDA and fabric manager

enrico-usai opened the pull request:
The latest version of Tesla driver is 470.82 that is not aligned to the latest Fabric manager version available (495.29).
We're upgrading the three packages to stay aligned with the latest version of drivers available, so:
* Nvidia and Fabric manager from 460.73 to 470.82
* CUDA from 11.3 to 11.4 (we're avoid the upgrade to 11.5 to keep driver version aligned)

See:
* https://docs.nvidia.com/datacenter/tesla/index.html
* https://developer.download.nvidia.com/compute/cuda/repos/rhel7/x86_64/

Note: the name of the Fabric Manager package in the repo has been changed.


Edit: CentOS8 has been deprecated by: https://github.com/aws/aws-parallelcluster-cookbook/pull/1259 so the following comments are no longer needed.

## ~~CentOS8 changes~~
~~In RHEL8, to be able to install a package version different from the latest one it is required to enable the module stream for that version. See:~~
* ~~NVIDIA/yum-packaging-fabric-manager#3~~
* ~~https://docs.fedoraproject.org/en-US/modularity/installing-modules/~~

~~To enable `nvidia-driver:470` stream and manage DFN modules on RHEL8+, the `dnf_module` cookbook resource is required. In this patch we're upgrading the yum package from 6.1.1 to [7.2.0](https://github.com/sous-chefs/yum/releases/tag/7.2.0) including the new `dnf_module` resource.~~

lukeseawalker commented on CHANGELOG.md:
```diff
@@ -3,6 +3,14 @@ aws-parallelcluster-cookbook CHANGELOG
+-----
+
+**CHANGES**
+- Upgrade NVIDIA driver to version 470.83.01.
```
470.82.01

enrico-usai replied:
fixed

lukeseawalker commented on attributes/default.rb:
```diff
@@ -131,13 +131,17 @@
+default['cfncluster']['nvidia']['cuda_url'] = 'https://developer.download.nvidia.com/compute/cuda/11.4.3/local_installers/cuda_11.4.3_470.82.01_linux.run'
+
+# The package name of Fabric Manager for alinux2 and centos7 is nvidia-fabric-manager-version
+# For ubuntu, it is nvidia-fabricmanager-470_version
```
For ubuntu is `nvidia-fabricmanager-<version>`

enrico-usai replied:
It is for example: nvidia-fabricmanager-470_470.82.01, 
I can change the comment to nvidia-fabricmanager-majorversion_version

enrico-usai replied:
done

enrico-usai commented:
~~It doesn't work on CentOS because of: https://github.com/NVIDIA/yum-packaging-fabric-manager/issues/3~~

~~Edit: patch updated by using dnf_module.~~

Edit: patch updated dropping CentoOS8

Commit:
Upgrade Nvidia Drivers, CUDA and fabric manager to 470.82

The latest version of Tesla driver is 470.82 that is not aligned to the latest
Fabric manager version available (495.29).

We're upgrading the three packages to stay aligned with the latest version of drivers available, so:
* Nvidia and Fabric manager from 460.73 to 470.82
* CUDA from 11.3 to 11.4 (we're avoid the upgrade to 11.5 to keep driver version aligned)
See:
* https://docs.nvidia.com/datacenter/tesla/index.html
* https://developer.download.nvidia.com/compute/cuda/repos/rhel7/x86_64/

Note: the name of the Fabric Manager package in the repo has been changed.

Signed-off-by: Enrico Usai <usai@amazon.com>
